### PR TITLE
[PT-BR] translating 'dt.ts' file for ptbr, from the app section

### PR DESCRIPTION
**Pull request by robertosousa1**
Hello world!
:)
@khaosdoctor, @alvarocamillont, @danilofuchs e @orta

### CODE REVIEW

File: packages/typescriptlang-org/src/copy/pt/dt.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+export const dtCopy = {
+  dt_s_page_title: "Procure por pacotes tipados",
+  dt_s_title: "Pesquisa de tipo",
+  dt_s_subtitle:
+    "Encontre pacotes npm que tenham declarações de tipo, agrupados ou Definitely Typed.",
```
Comment (khaosdoctor) [inline packages/typescriptlang-org/src/copy/pt/dt.ts]:
```suggestion
    "Encontre pacotes npm que tenham declarações de tipo, tanto agrupados quanto no Definitely Typed.",
```

File: packages/typescriptlang-org/src/copy/pt/dt.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  dt_s_match: "resultado",
+  dt_s_matchs: "resultados",
+  dt_s_match_exact: "Combinação exata",
+  dt_s_popular_on_dt: "Popular em Definitely Typed",
```
Comment (khaosdoctor) [inline packages/typescriptlang-org/src/copy/pt/dt.ts]:
```suggestion
  dt_s_popular_on_dt: "Popular no Definitely Typed",
```

File: packages/typescriptlang-org/src/copy/pt/dt.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  dt_s_downloads_short: "DLs",
```
Comment (khaosdoctor) [inline packages/typescriptlang-org/src/copy/pt/dt.ts]:
```suggestion
  dt_s_downloads_short: "Downl.",
```

File: packages/typescriptlang-org/src/copy/pt/dt.ts [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  dt_s_downloads_via: "Via",
+  dt_s_module: "Módulo",
+  dt_s_last_update: "Última atualização",
+  dt_s_install: "Instalar",
+  dt_s_no_results: "Nenhum resultado encontrado para",
+  dt_s_no_results_try: "Tente outra pesquisa?",
```
Comment (khaosdoctor) [inline packages/typescriptlang-org/src/copy/pt/dt.ts]:
```suggestion
  dt_s_no_results_try: "Tentar outra busca?",
```

**Review by khaosdoctor — CHANGES_REQUESTED** [review]

**Comment by khaosdoctor** [comment]
@robertosousa1 If you can, please correct these changes until january 25th (See #233 for information) so we can merge it and migrate to the new repo.

Thanks!

**Comment by robertosousa1** [comment]
Hi @khaosdoctor, changes made as requested, any eventuality, I am available.

Thanks!

**Review by khaosdoctor — APPROVED** [review]

**Comment by khaosdoctor** [comment]
lgtm

**Comment by khaosdoctor** [comment]
@robertosousa1 can you please fix the merge conflicts?

**Comment by robertosousa1** [comment]
Hi @khaosdoctor.

sorry, but I couldn't find the conflicts to resolve them.

only "This branch has no conflicts with the base branch" appears to me, I tried to merge directly via vscode via my fork, but the conflicts did not appear either.

could you help me please?

**Comment by khaosdoctor** [comment]
lgtm

**Review by khaosdoctor — APPROVED** [review]

**Comment by khaosdoctor** [comment]
Done!

**Pull request merged**